### PR TITLE
Added: issue fixes 326-330

### PR DESCRIPTION
**Pull request by nick-samygin**
I've checked the following points:

- [ ] I've formatted the code by pressing (STRG + K + D)
- [ ] I've removed unnecessary usings
- [ ] I've not used magic variables
- [ ] I've used localization service instead of text
- [ ] All `public` things have comments
- [ ] This has been followed: [conventions](https://simplic.github.io/dev/csharp/getting_started/conventions.html)

### CODE REVIEW

File: simplic-user/Simplic.User.UI/Utils/IOpenSelectedUserDetails.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+namespace Simplic.User.UI.Utils
+{
+    interface IOpenSelectedUserDetails
+    {
+        ICommand OpenSelectedUserDetailsCommand
+        {
+            get;
+            set;
+        }
+    }
```
Comment (simplicbe) [inline simplic-user/Simplic.User.UI/Utils/IOpenSelectedUserDetails.cs:12]: Interfaces should have a comment in any way

File: simplic-user/Simplic.User.UI/ViewModels/UserViewModel.cs [hunk 2]
```diff
@@ -120,7 +120,20 @@ public void SaveUser()
...
+                User.IsADUser = IsADUser;
+                User.KeepLoggedIn = KeepLoggedIn;
+                User.LanguageID = LanguageID;
+                User.Password = Password;
+                User.UserName = UserName;
                 _userService.Save(User);
+            }
         }
 
         public void SavePassword()
```
Comment (simplicbe) [inline simplic-user/Simplic.User.UI/ViewModels/UserViewModel.cs:139]: Everything that is public needs a comment

**Comment by simplicbe** [comment]
@nick-samygin please rebase the branch from dev.

**Comment by simplicbe** [comment]
@nick-samygin please resolve the conflicts

File: simplic-user/Simplic.User.UI/Utils/IOpenSelectedUserDetails.cs (hunk 1 above)
Comment (nick-samygin) [reply simplic-user/Simplic.User.UI/Utils/IOpenSelectedUserDetails.cs:12]: Done

**Pull request merged**